packages/server/test/PostHandler.test.ts

```diff
@@ -24,7 +25,10 @@ describe('PostHandler', () => {
   fake_store.hasExtension.withArgs('creation-defer-length').returns(true)
 
   beforeEach(() => {
-    req = {url: '/files', method: 'POST'} as http.IncomingMessage
+    req = new MockIncomingMessage({
+      url: '/files',
+      method: 'POST',
+    }) as unknown as http.IncomingMessage
```

Comment:
I think we can replace this with `node-mocks-http`, which we already use:

https://github.com/tus/tus-node-server/blob/612ac24cf7507ddeda57cab90f3bc95d9dfbcb6a/packages/server/test/DeleteHandler.test.ts#L24